packages/app-mining/src/pages/DelegateV2.tsx

```diff
@@ -1,11 +1,18 @@
-import {Helmet} from 'react-helmet'
+import {useInterval} from '@phala/react-hooks'
+import {Block} from 'baseui/block'
 import {Card} from 'baseui/card'
+// import {useState} from 'react'
+import {Helmet} from 'react-helmet'
 import DelegateBanner from '../components/DelegateBanner'
 import StakePoolTableV2 from '../components/StakePoolTableV2'
 import StatusBar from '../components/StatusBarV2'
-import {Block} from 'baseui/block'
 
 export const DelegateV2 = (): JSX.Element => {
+  // const [pollingCount, setPollingCount] = useState(0)
+  useInterval(() => {
+    window.location.reload()
```

Comment:
Reload page is a bad practice, it may interrupt user's actions.

Actually we are using react-query, so `refetchInterval` is a good feature for this case.

Reply:
thx, got it